packages/playground/src/components/networks.vue

```diff
@@ -159,29 +115,21 @@ export default {
     ) {
       throw new Error("You must provide at least one network  option");
     }
-    const error = computed(
-      () => !(props.ipv4 || props.ipv6 || props.planetary || props.wireguard || props.mycelium) && props.required,
-    );
-    expose({
-      error,
+    const error = computed(() => {
```

Comment:
on k8s  I was able to disable all networks and click on deploy without any errors 
![image](https://github.com/user-attachments/assets/00a07125-9bfb-43cf-987b-0221903a8674)